### DISCUSSION TITLE
Matomo & Breeze

### DISCUSSION THREAD

**Discussion by 2EyeTec** [breezefront/community#143] [1. Help] [answered]
I'm trying to get Matomo tracking my webshop. I can see the js code in Firebug, but I don't see the POST request with matomo.php. When I disable Breeze with ?breeze=0 I can see the POST request and my visit is tracked.

Thats the module I'm using: https://github.com/fnogatz/magento2-matomo

The url is www.2eyetec.com

What do I have to do to make it work WITH breeze enabled?

Thanks!

**Comment by vovayatsyuk** [comment]
You need to integrate all js files. 

This module uses one on the front: https://github.com/fnogatz/magento2-matomo/blob/main/view/frontend/web/js/tracker.js
Additionally, it uses inline scripts: https://github.com/fnogatz/magento2-matomo/blob/main/view/frontend/templates/matomo.phtml

**Comment by 2EyeTec** [comment, marked as answer]
I put the Matomo Tracking code in the Footer and now Matomo is tracking my website again! First I tried to put it into the header, but this did not work.
I disabled the "offical" matomo module in the backend.
